### ISSUE TITLE
Edit New issue As an erised admin, I'd like players to have to agree to terms before they can do certain things.

### ISSUE THREAD

**Issue by fuzzyweapon** [erised/erised#8]
things you can and cannot do before agreeing...
-should be able to talk on some sort of guest channel
-shouldn't be able to get on any IC channels of course
-shouldn't be able to spam other users (in the form of ads, messages/inbox, channels)
-shouldn't gate someone from working on chargen, but we should remind the user if they have not agreed yet.

-once they agree, it is final, timestamped, +note'd on the player object
-once they agree, they are allowed into an OOC area where they can interact with anyone on the game "in-person".
I suspect OOC will be more meaningful here since people have a Player account.

MVP DoD:
- [] player can agree
- [] agreement gates @ic

**Comment by HeroicLeigh** [comment]
I agree with this. Especially the being able to post ads. I hate when people create on a game, just to post an ad. I don't care if they post the game's ad elsewhere- it's just rude, IMO, to advertise on a game you don't even play on.
